### DISCUSSION TITLE
Tapping `M` and `L` at the initial cookie should bring you to multi and playlists

### DISCUSSION THREAD

**Discussion by Susko3** [ppy/osu#17854] [Ideas] [answered]
Same as tapping `P` three times will bring you to solo play.
Currently, you have to be in the 'play' sub-menu for `M` and `L` to work.

If you agree with this behaviour, I have a PR ready.

**Comment by bdach** [comment]
I don't know if it "should". Also the transition needs to look good.

(Also I believe that the main menu is also going to get redesigned at some point so... possibly wasted effort)

**Reply by Susko3** [reply]
The transition is the same as the current transition whey pressing `P` thrice, except that multi/playlists are activated as the last press.

> Also I believe that the main menu is also going to get redesigned at some point so...

If we want this behaviour, best to implement it now, so we don't forget to implement it in the new main menu.

**Reply by bdach** [reply]
So why should `P` need to be pressed three times to go to solo, but `M` should work immediately?

**Reply by Susko3** [reply]
I meant pressing it three times to get to multi, same as with `P`.

On the initial, cookie only screen, pressing any of the keys will first press the cookie, then the play menu will open, and only then is solo/multi/playlists activated.

**Reply by Susko3** [reply]
On the initial screen, pressing any of the keys from one level down activates the cookie, so it makes sense that pressing keys one level down from 'Play' should activate it. And since there are no conflicts, why shouldn't those keys also activate the cookie?

**Reply by peppy** [reply]
The main menu key binds are there with the intention of being the initials of controls which are visible on the screen. So I would be against "m" going into the "play" section, but I am not against making the main menu "initial" mode (where only the osu! cookie is visible) react to any and every key press (rather than only valid ones that would display after it expands to the top level menu).

**Comment by Susko3** [comment, marked as answer]
Resolved by https://github.com/ppy/osu/pull/17950
